src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -444,6 +488,42 @@ public static void writeBundle(
                                         + name);
                     }
                 }
+
+                // process for async components
+                if (!components.stream()
+                                .filter(c -> !c.canBeGeneratedAsync())
+                                .toList()
+                                .isEmpty()
+                        && outputPath != null) {
+                    try {
+                        File zipFile = outputPath.resolve(SYNC_SUPPORT_BUNDLE).toFile();
+                        try (ZipInputStream zis = new ZipInputStream(new FileInputStream(zipFile))) {
+                            ZipEntry entry;
+                            while ((entry = zis.getNextEntry()) != null) {
+                                if (entry.getName().equals("manifest.md")) {
+                                    // no need to add the manifest.md as it will be created in the async flow
+                                    continue;
+                                }
+
+                                binaryOut.putNextEntry(entry);
+
+                                // Copy the content
+                                byte[] buffer = new byte[4096];
+                                int bytesRead;
+                                while ((bytesRead = zis.read(buffer)) != -1) {
+                                    binaryOut.write(buffer, 0, bytesRead);
+                                }
+
+                                binaryOut.flush();
+                            }
+                        }
+                    } catch (Exception e) {
+                        LOGGER.warning("Error while processing sync components in asyn mode: " + e.getMessage());
```

Comment:
```suggestion
                        LOGGER.warning("Error while processing sync components in async mode: " + e.getMessage());
```
or actually do not swallow stack traces
```suggestion
                        LOGGER.log(Level.WARNING, "Error while processing sync components in async mode", e);
```